src/server.js

```diff
@@ -155,11 +166,13 @@ Server.prototype.receiveEdit = function(connection, editMessage, sendToClient){
         if(!isEmpty(edit.diff)){
           clientDoc.shadow.localVersion++;
         }
+
+        // notify about patch
+        this.emit('patch', { clientDoc, connection, edit, editMessage });
       }else{
         // TODO: implement backup workflow
         // has a low priority since `packets are not lost` - but don't quote me on that :P
-        console.log('error', 'patch rejected!!', edit.serverVersion, '->', clientDoc.shadow.serverVersion, ':',
-                    edit.localVersion, '->', clientDoc.shadow.localVersion);
+        this.emit('patch_rejected', { clientDoc, connection, edit, editMessage });
```

Comment:
I love ES6 and use it in all my projects at the moment but when we decide to use it we need to either publish these changes in a new major release (due to breaking changes for node <= 0.12 users) or provide a build step before we publish it to NPM.

What do you prefer?